README.md

```diff
@@ -1 +1,3 @@
-# myFlix-client
\ No newline at end of file
+# myFlix-client
+
+parcel [Path to index.html]
```

Comment:
you can write here the right command you need to start your local server like so:

## Start Server
`parcel src/index.html`